pkg/crc/validation/validation.go

```diff
@@ -61,18 +61,7 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 	}
 
 	userProvidedBundle := filepath.Base(bundlePath)
-	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
-	if userProvidedBundle != constants.GetDefaultBundle(preset) {
-		// Should append underscore (_) here, as we don't want crc_libvirt_4.7.15.crcbundle
-		// to be detected as a custom bundle for crc_libvirt_4.7.1.crcbundle
-		usingCustomBundle := strings.HasPrefix(bundle.GetBundleNameWithoutExtension(userProvidedBundle),
-			fmt.Sprintf("%s_", bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle(preset))))
-		if usingCustomBundle {
-			logging.Warnf("Using custom bundle %s", userProvidedBundle)
-			return nil
-		}
-		logging.Warnf("Using %s bundle, but %s is expected for this release", userProvidedBundle, constants.GetDefaultBundle(preset))
-	}
```

Comment:
> We also don't want user to show both warning in case user uses custom bundle, so this patch also logs only respective warning as per bundle filename or metadata.

The code before adding the helper was not printed 2 warnings, but just one. The quoted part of the commit log seems to emphasize that there is no behaviour change, but I don't think it's very important to have it.